Formula/vaulted.rb

```diff
@@ -0,0 +1,43 @@
+class Vaulted < Formula
+  desc "Allows the secure storage and execution of environments"
+  homepage "https://github.com/miquella/vaulted"
+  url "https://github.com/miquella/vaulted/archive/v2.0.tar.gz"
+  sha256 "7bf3e9dc0ae015f1fadee3520273707ad072a2f6c945c959c993d282301c897e"
+
+  head "https://github.com/miquella/vaulted"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    # Install Go dependencies
+    system "go", "get", "github.com/aws/aws-sdk-go/aws"
```

Comment:
`go get` is pretty much never desired in a Homebrew formula. Anywhere it still exists it shouldn't, and definitely shouldn't be adding more of it 😅.

In an ideal world upstream would be using the vendor system, but otherwise this will need to use `go_resource`s. See [gx](https://github.com/Homebrew/homebrew-core/blob/09f1387e35944ce45d376f40a69aa034d8a0b994/Formula/gx.rb) for an example. Sometimes [gdm](https://github.com/sparrc/gdm) can be useful for implementing that system semi-automatically.